exporter/doc/id.go

```diff
@@ -0,0 +1,74 @@
+package doc
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/projectcypress/cdatools/models"
+)
+
+type CDAIdentifier struct {
+	models.CDAIdentifier
+}
```

Comment:
The goal of these types in the `doc` package is for them to represent the new templates. We can print these documents represented by these names as our templates, independent of their original model.